src/lucky/memoizable.cr

```diff
@@ -22,7 +22,7 @@ module Lucky::Memoizable
   macro memoize(method_def)
     {% raise "You must define a return type for memoized methods" if method_def.return_type.is_a?(Nop) %}
     {%
-      raise "All arguments must have an explicit type for memoized methods" if method_def.args.any? &.is_a?(Nop) # ameba:disable Style/IsAFilter
+      raise "All arguments must have an explicit type for memoized methods" if method_def.args.any? &.is_a?(Nop)
```

Comment:
I can break this out into a separate PR after we work through the rest of the stuff, but I wanted to get the CheckFormat specs passing while we're tinkering.